app/src/main/java/io/homeassistant/companion/android/sensors/PhoneStateSensorManager.kt

```diff
@@ -82,6 +92,15 @@ class PhoneStateSensorManager : SensorManager {
             phoneIcon,
             mapOf()
         )
+
+        if (isEnabled(context, callNumber.id) && currentPhoneState == "idle") {
+            onSensorUpdated(context,
```

Comment:
This does not trigger an update to get sent to HA, it only updates the DB here. If the state changes next time we send an update it will send to HA.

Reply:
Yes, like for any other sensors

Reply:
Yea the line numbers were playing tricks on me 🙈 sorry about that